--- /home/owner/git/EnlivenMinetest/Bucket_Game-base/vbutterfly_license_fix-vs-220429/mods/codermobs/codermobs/LICENSE	2022-05-01 15:44:14.808012258 -0400
+++ /home/owner/git/EnlivenMinetest/Bucket_Game-branches/vbutterfly_license_fix-vs-220429/mods/codermobs/codermobs/LICENSE	2022-05-01 15:44:49.264011773 -0400
@@ -206,8 +206,7 @@
 
 Butterfly (voxel art) textures
 -----------------------------
-unknown
-- please submit author info to Poikilos or OldCoder
+CC BY-SA 3.0 (C) 2016 D00Med (for LegendofMinetest mapgen)
 
 Realistic Milk Bucket
 -----------------------------
